sunpy/timeseries/timeseriesbase.py

```diff
@@ -461,7 +461,7 @@ def plot(self, axes=None, **plot_args):
 
```

Comment:
I can't highlight the line, but it says above the `**plot_args` are passed to :meth:`pandas.DataFrame.plot`. But is that true?

Reply:
`self._data` returns a `pandas.core.frame.DataFrame`, so arguments passed to `self._data.plot()` are sent to pandas